config/samples/numaplane.numaproj.io_v1alpha1_isbservicerollout.yaml

```diff
@@ -9,3 +9,5 @@ spec:
       # Example from https://github.com/numaproj/numaflow/blob/main/examples/0-isbsvc-jetstream.yaml
       jetstream:
         version: 2.10.3
+        persistence:
+          volumeSize: 1Gi
```

Comment:
I accidentally removed this from here when my real intention was to remove it from the test manifests - restoring